pygmt/datasets/load_remote_dataset.py

```diff
@@ -592,11 +592,12 @@ def _load_remote_dataset(
                 kind=dataset.kind, outgrid=None, vfname=voutgrd
             )
 
-    # Full path to the grid if not tiled grids.
-    source = which(fname, download="a") if not resinfo.tiled else None
+    # Full path to the grid
+    source: str | list = which(fname, download="a", verbose="q")
```

Comment:
Hmmm. Actually, this won't work, since `which("@earth_relief_03s", download="a", verbose="q")` will download all tiles, which take forever.